desk/app/groups.hoon

```diff
@@ -2,21 +2,39 @@
 /-  g-one=group
 /-  m-one=metadata-store
 /-  meta
+/-  e=epic
 /+  default-agent, verb, dbug
 /+  groups-json  :: unused, nice for perf
 /+  of
 /*  desk-bill  %bill  /desk/bill
 ^-  agent:gall
 =>
   |%
+  ++  okay  ^-  epic:e  0
   +$  card  card:agent:gall
   +$  state-0
     $:  %0
         groups=(map flag:g [net:g group:g])
         xeno=(map flag:g gang:g)
     ==
+  +$  state-1
+    $:  %1
+        cool=epic:e
+        groups=(map flag:g [net:g group:g])
+        xeno=(map flag:g gang:g)
+    ==
+  +$  versioned-state  $%(state-0 state-1)
+  ::
+  ++  jack
+    |=  v=versioned-state
+    ?-  -.v
+      %0  [%1 cool=*epic:e +.v]
```

Comment:
This doesn't fully migrate because you need to migrate `%net`

Reply:
ah right, will fix.